test/test_fx_experimental.py

```diff
@@ -7,9 +7,11 @@
 import sys
```

Comment:
ideally we can put this test file in a sub folder like `test/fx/`

Reply:
Moved.